frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/pvc.scenario.ts

```diff
@@ -1,30 +1,36 @@
+import { execSync } from 'child_process';
 import { browser } from 'protractor';
-import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
 import { resourceRowsPresent } from '@console/internal-integration-tests/views/crud.view';
 import {
+  addStorage,
   createNewPersistentVolumeClaim,
   deletePersistentVolumeClaim,
   goToPersistentVolumeClaims,
   pvcStatus,
   pvcSize,
 } from '../../views/pvc.view';
 import {
-  NS,
   PVC_STATUS,
   SIZE_UNITS,
   STORAGE_CLASS_PATTERNS,
   VOLUME_ACCESS_MODES,
 } from '../../utils/consts';
+import {
+  deployment,
+  testDeployment,
+} from '@console/noobaa-storage-plugin/integration-tests/mocks/deploymentData';
```

Comment:
Export the `deployment` and `testDeployment` from ceph. We have ceph package dependent on noobaa and not the reverse. Lets stick to that and be saved from cyclic dependencies.
Thanks :)

Reply:
Thanks! Done